packages/httpsnippet-client-api/test/__datasets__/multipart-data/har.json

```diff
@@ -17,7 +17,7 @@
               {
                 "name": "foo",
                 "value": "Hello World",
-                "fileName": "hello.txt",
+                "fileName": "test/__fixtures__/files/hello.txt",
```

Comment:
Changed this file path so that the `hello.txt` fixture I added in this PR will get picked up and sent with the snippet.